src/ast.hpp

```diff
@@ -279,48 +324,50 @@ namespace Sass {
   template <typename T>
   inline Vectorized<T>::~Vectorized() { }
 
+
+
   /////////////////////////////////////////////////////////////////////////////
   // Mixin class for AST nodes that should behave like a hash table. Uses an
   // extra <std::vector> internally to maintain insertion order for interation.
   /////////////////////////////////////////////////////////////////////////////
   class Hashed {
   struct HashExpression {
-    size_t operator() (Expression* ex) const {
+    size_t operator() (Expression_Obj ex) const {
       return ex ? ex->hash() : 0;
     }
   };
   struct CompareExpression {
-    bool operator()(const Expression* lhs, const Expression* rhs) const {
+    bool operator()(const Expression_Obj lhs, const Expression_Obj rhs) const {
       return lhs && rhs && lhs->eq(*rhs);
     }
   };
   typedef std::unordered_map<
-    Expression*, // key
-    Expression*, // value
+    Expression_Obj, // key
+    Expression_Obj, // value
     HashExpression, // hasher
     CompareExpression // compare
-  > ExpressionMap;
+  > ExpressionMap2;
```

Comment:
Fixed. Used during dev since I had to replace each class with an updated one without breaking stuff.